_pages/algorithm-development/source-control-management.md

```diff
@@ -0,0 +1,154 @@
+{: .notice-info}
+
+**Web IDE Support** At this time we do not support editing source code in our web app for GitHub-hosted algorithms. 
+{: .notice-info}
+
+#### Connecting Your Algorithmia & GitHub Accounts
+
+To connect your Algorithmia and GitHub accounts, simply select the GitHub instance you wish to use when creating your algorithm:
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/scm_create_github_algorithm_unauthorized.png" alt="Authorization prompt for GitHub repository host" class="screenshot img-sm">
+
+If you haven't yet connected your GitHub account, you will be prompted to do so. By connecting your GitHub account, you will give us access to create repositories on your behalf, which will house your algorithm's source code.
+
+#### Creating Algorithms with GitHub
+
+Once you've connected your GitHub account to Algorithmia, you will be able to host algorithm source code in GitHub.
+
+You can customize two aspects of the GitHub repositories that are created for your algorithms: the repository's owner, and its name.
+
+By default, all GitHub respositories are created under your personal GitHub account. However, you may optionally choose any GitHub organizations that you are a member of. These organizations are listed in the "Repository Owner" dropdown:
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/scm_create_github_algorithm_org.png" alt="Customizing your repository owner" class="screenshot img-sm">
+
+You may also customize the name of the repository that is created. By default the name of your algorithm is used. 
+
+<img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/source_code_management/scm_create_github_algorithm_name.png" alt="Customizing your repository name" class="screenshot img-sm">
+
+If you decide to use an alternative name, we recommend that you use only letters, numbers, hyphens, and/or underscores. While GitHub will accept characters outside this range, they will replace any unsupported characters with an underscore, resulting in an undesirable repository name.
+
+When we create your repository, we associate the following with it:
+
+- **A Deploy Key**: [GitHub deploy keys](https://github.blog/2015-06-16-read-only-deploy-keys/) allow read-only access to specific repositories, and are GitHub’s prescribed means by which external services can fetch code for building and deploying. These keys are not tied to individual permissions, and as such will allow Algorithmia to continue building an algorithm even if the permissions of the creating user change.
+- **Webhooks**: We set up webhooks to receive notifications about changes to your repo, such as a commit to its master branch, or when its name change.
```

Comment:
do we want to specify that we look at the default branch which doesn't need to be master? 

And if we get the hmac secret stuff done, is this where we would update the docs? (Hoping to have that out for review soon)

Reply:
Will add!